diff --git a/node_modules/@expo/cli/build/src/run/ios/appleDevice/client/LockdowndClient.js b/node_modules/@expo/cli/build/src/run/ios/appleDevice/client/LockdowndClient.js
index 4005594..9f23642 100644
--- a/node_modules/@expo/cli/build/src/run/ios/appleDevice/client/LockdowndClient.js
+++ b/node_modules/@expo/cli/build/src/run/ios/appleDevice/client/LockdowndClient.js
@@ -112,7 +112,7 @@ class LockdowndClient extends _ServiceClient.ServiceClient {
         }
     }
     async startSession(pairRecord) {
-        debug(`startSession: ${pairRecord}`);
+        debug('startSession: %o', pairRecord);
         const resp = await this.protocolClient.sendMessage({
             Request: 'StartSession',
             HostID: pairRecord.HostID,
